pyquil/device.py

```diff
@@ -330,7 +330,7 @@ def __init__(self, name, raw):
         self._raw = raw
         self.isa = ISA.from_dict(raw['isa']) if 'isa' in raw and raw['isa'] != {} else None
         self.specs = Specs.from_dict(raw['specs']) \
-            if 'specs' in raw and raw['specs'] != {} else None
```

Comment:
The whole line can just be:
`if raw.get('specs', None)`, both None and empty dictionary are False in Python